gcs/bench_test.go

```diff
@@ -88,15 +95,15 @@ func BenchmarkFilterMatch(b *testing.B) {
 	}
 
 	var key [KeySize]byte
-	filter, err := NewFilterV1(P, key, contents)
+	filter, err := NewFilterV2(benchB, benchM, key, contents)
 	if err != nil {
 		b.Fatalf("Failed to build filter")
 	}
 
 	b.ReportAllocs()
 	b.ResetTimer()
 	for i := 0; i < b.N; i++ {
-		globalMatch = filter.Match(key, []byte("Nate"))
+		globalMatch = filter.Match(key, []byte("Something"))
```

Comment:
curious about this change

Reply:
The way that GCS filters work is that the very first entry is the difference between the reduced hash value and zero.  The result is that a single entry can vary wildly in the benchmarks due to the unary encoding of it.  That particular value just happened to be a perfect case for v1 filters and the worst possible case for v2 filters.  This change results in a fair value for both.

I called this out in the PR description with the bit about:

> It is interesting to note the results for attempting to match a single item is not very representative due to the fact the actual hash value itself dominates to the point it can significantly vary due to the very low ns timings involved